io/plugins/eu.esdihumboldt.hale.io.xtraserver.test/src/eu/esdihumboldt/hale/io/xtraserver/writer/XtraServerMappingFileWriterTest.java

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (c) 2015 Data Harmonisation Panel
```

Comment:
The IPR transfer was completed end of last year, so yes, we should update the headers to reflect that wetransform is the IPR holder. As per other contributors, II would appear as the contributor, but not as the copyright holder, in the header of these files.

Reply:
> As per other contributors, II would appear as the contributor, but not as the copyright holder, in the header of these files.

@azahnen This is not correct. The ii copyright is untouched by the contributors agreement. So no need to change anything on that behalf.

Reply:
Simon is correct, I confused the clauses in the IPR transfer agreement and in the CA.

Reply:
@stempler Yes, we included the dhp copyright because the file is based on a copy from the appschema plugin. So, should I leave the copyright as it is or change it to wetransform or ii?

Reply:
I guess the simplest solution might be using [AbstractProjectTest](https://github.com/halestudio/hale/blob/master/common/plugins/eu.esdihumboldt.hale.common.headless.test/src/eu/esdihumboldt/hale/common/headless/test/AbstractProjectTest.java) to remove the project loading logic copied from the original file and use the ii copyright then.

Reply:
Ok, I will do that.